12-offer-encoding.md

```diff
@@ -362,6 +361,7 @@ invoices is `lnr`.
 ## Requirements for Invoice Requests
 
 The writer of an invoice_request:
+  - SHOULD set `payer_info` to an unpredictable series of bytes.
```

Comment:
If you don't care about privacy?  Should we say MUST?

I set this to the data I need to recover the payer_key from my node_id, which (1) means I don't need to remember it, but can reconstruct from any invoice and (2) it's unpredictable to anyone else.

Reply:
I meant populating the `payer_info` in general, rather than the value that we have in it: 
- MUST set `payer_info` to a non-zero series of bytes
  - SHOULD set `payer_info` to an unpredictable series of bytes

Reasoning being that if we're going to use `first-tlv` in nonce calculation, not setting `payer_info` (which this wording allows) would mean you use the predictable `offer_chains` tlv?

Reply:
(somewhat relevant to [this comment](https://github.com/rustyrussell/lightning-rfc/pull/7/files#r948137786) about specifying using `invoice_request_payer_info` rather than just "hoping it's there")